error[E0225]: only auto traits can be used as additional traits in a trait object
  --> $DIR/E0225.rs:17:32
   |
LL |     let _: Box<std::io::Read + std::io::Write>; //~ ERROR E0225
   |                                ^^^^^^^^^^^^^^ non-auto additional trait

error[E0225]: only auto traits can be used as additional traits in a trait object
  --> $DIR/E0225.rs:18:16
   |
LL | trait Foo = std::io::Read + std::io::Write;
   |                             -------------- non-auto additional trait
LL | trait Bar = Foo;
   |             --- referenced by this alias
...
LL |     let _: Box<Bar>; //~ ERROR E0225
   |                ^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0225`.
